.github/workflows/causal_lm_cpp.yml

```diff
@@ -774,6 +774,7 @@ jobs:
         run: |
           source ./ov/setupvars.sh
           timeout 30s ./build/samples/cpp/text_generation/encrypted_model_causal_lm ./TinyLlama-1.1B-Chat-v1.0/ "Why is the sun yellow?"
+          timeout 30s  python ./samples/python/text_generation/encrypted_model_causal_lm.py ./TinyLlama-1.1B-Chat-v1.0/ "Why is the sun yellow?"
```

Comment:
Verify that the outputs are the same. I think it's best to use `tee` so we see the full output and `diff` after that

Reply:
https://github.com/openvinotoolkit/openvino.genai/blob/93f305de93f388ea171b2a816e3a65ac5f52b60b/.github/workflows/causal_lm_cpp.yml#L866

Reply:
Sample tests moved to pytest in https://github.com/openvinotoolkit/openvino.genai/pull/1661. Please, do the same

Reply:
done